src/ui/units/wizard/containers/Wizard/SectionVisualization/PlaceholdersContainer/ColorsPlaceholder/ColorsPlaceholder.tsx

```diff
@@ -94,10 +94,12 @@ class ColorsPlaceholder extends React.Component<Props> {
 
     private onColorsUpdate = (items: Field[]) => {
         this.props.updateColors({items});
-        this.props.updatePreviewAndClientChartsConfig({});
 
         if (this.props.onUpdate) {
             this.props.onUpdate();
+        } else {
+            // Else call default onUpdate action
+            this.props.updatePreviewAndClientChartsConfig({});
```

Comment:
Here too
This is a very non-obvious behavior. I suggest you return it as it was